packages/material-ui/src/Breadcrumbs/Breadcrumbs.js

```diff
@@ -60,6 +116,12 @@ const Breadcrumbs = React.forwardRef(function Breadcrumbs(props, ref) {
     ...other
   } = props;
 
+  const styleProps = {
+    ...other,
```

Comment:
```suggestion
    ...props,
```

plus please add all defaults: `component` etc..